tests/test_python_emulator.py

```diff
@@ -159,3 +161,87 @@ def test_serialization():
     obj_str = dumps(batch)
     batch2 = loads(obj_str)
     assert isinstance(batch2, type(batch))
+
+
+def KS_test(
```

Comment:
is there a package for this? or we are gonna maintain this

Reply:
looks like there is am implementatoin in scipy: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ks_2samp.html#scipy.stats.ks_2samp